LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, QM/MM Study Group
```

Comment:
Can this be changed to `Copyright (c) 2018, QM/MM Study Group (See AUTHORS.md)` and we add an AUTHORS.md with a list of authors. Sorry for the trouble